### ISSUE TITLE
Tmuxifier creates server as x86_64 even though tmux on its own creates server as arm64

### ISSUE THREAD

**Issue by ahollister** [jimeh/tmuxifier#89]
I reinstalled tmux and homebrew this morning on my M1 Macbook Pro to ensure they are using the arm64 architecture.

In my ZSH shell (outside tmux) I get this output for uname:

```
$ uname -m
arm64
```

Checking the tmux binary I get this output:

```
$ file $(which tmux)
/opt/homebrew/bin//tmux: Mach-O 64-bit executable arm64
```

If I run tmux (when no server exists) and check uname again:

```
$ uname -m
arm64
```

If I kill the tmux server and start a new tmux server by running a tmuxifier command, the session I land in will be x86_64:

```
$ uname -m
x86_64
```

However, if I kill the tmux server, start a fresh one with the tmux command, THEN use tmuxifier to create a session, that session will be in an arm64 context.

So it appears that starting a tmux server by using a tmuxifier command will create an x86_64 server, even though the tmux command knows to use arm64.

This causes a bunch of compatibility errors in treesitter in neovim.

**Comment by ahollister** [comment]
hm yeah you seem to be right. I get x86_64, guess that means I should install an arm64 version of bash and default to that?

**Comment by ahollister** [comment]
ah it seems I have two versions of bash on my machine, one is 5.2 and x86, the other is 3.2 and arm64

**Comment by ahollister** [comment]
OK I figured out what happened.

When I set up this computer about a year ago I must have used a backup of my older Intel mac instead of a clean install, so I ended up with lots of x86 packages including bash and brew.

The solution in the end was to use `brew bundle` to generate a `Brewfile`, move the contents of `/usr/local` into a directory called `BACKUP`, re-install brew from scratch and reinstall packages from the Brewfile.
